WebAppForSolocoProject/Controllers/HomeController.cs

```diff
@@ -45,36 +48,24 @@ public async Task<IActionResult> Create()
         public async Task<IActionResult> Run(HomeCreateVM model)
         {
             await UpdateModel(model);
+            DeleteAppConfig("app.config");
+            RenameAppConfig("original_app.config", "app.config");
             return View("Create", model);
         }
 
         public async Task<HomeCreateVM> CreateDirectories(HomeCreateVM model)
         {
-            if (model.BasePath == null)
+            await Task.Run(() =>
             {
-                model.Logs.Add("You must provide a base path!");
-            }
-            else
-            { 
-                try
+                foreach (var key in model.SelectedOwner.FolderPathPairs.Keys)
                 {
-                    await Task.Run(() =>
+                    foreach (var path in model.SelectedOwner.FolderPathPairs.GetValueOrDefault(key))
                     {
-                        foreach (var key in model.SelectedOwner.FolderPathPairs.Keys)
-                        {
-                            foreach (var path in model.SelectedOwner.FolderPathPairs.GetValueOrDefault(key))
-                            {
-                                string pathToCreate = Path.Combine(model.BasePath, model.SelectedOwnerName, path);
-                                model.Logs.Add(CreateDirectory(pathToCreate));
-                            }
-                        }
-                    });
-                }
-                catch (UnauthorizedAccessException)
-                {
-                    model.Logs.Add("You don't have access in selected directory. Please change your base path.");
+                        string pathToCreate = Path.Combine(model.BasePath, model.SelectedOwnerName, path);
```

Comment:
this var can be removed since is used only in one place... and path.combine could be added inside  call of  CreateDirectory